modules/features/manatime_feature_timesheets/manatime_feature_timesheets.module

```diff
@@ -279,12 +292,29 @@ function manatime_feature_timesheets_remove_row_form($form, &$form_state) {
 }
 
 /**
+ * Timesheet submit form.
+ */
+function manatime_feature_timesheets_submit_form($form, &$form_state) {
```

Comment:
Try getting $node and set a form element of type => value in order to later get it from $form_state['values']